kasa/cli.py

```diff
@@ -1219,18 +1219,30 @@ async def feature(dev: Device, child: str, name: str, value):
     If only *name* is given, the value of named feature is returned.
     If both *name* and *value* are set, the described setting is changed.
     """
+
+    def _print_feature(name, feat, indent=""):
+        try:
+            unit = f" {feat.unit}" if feat.unit else ""
+            if feat.type == Feature.Type.Choice:
+                vals = feat.choices
+                index = vals.index(feat.value)
+                vals.remove(feat.value)
+                vals.insert(index, f"*{feat.value}*")
+                val = " ".join(vals)
```

Comment:
The formatting should go into `Feature.__repr__`, and it'd be good if printing the features would use `_echo_features` for consistency.

Reply:
Done